### PR TITLE
add tabs additional css

### PR DESCRIPTION
**Pull request by zombieJ**
ref: #13086

### CODE REVIEW

**Comment by zombieJ** [comment]
done. pls check.

**Comment by afc163** [comment]
本地有测过各种内嵌的情况么，搞个 debug: true 的 demo 吧。

ink-bar 是不是也要加 prefix

**Comment by afc163** [comment]
master 的 ci 要修一下

**Comment by afc163** [comment]
oops

**Comment by zombieJ** [comment]
move to `feature` branch.

File: components/tabs/TabBar.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import Icon from '../icon';
```
Comment (ilanus) [inline components/tabs/TabBar.tsx]: Absolute imports should come before relative imports

File: components/tabs/TabBar.tsx [hunk 2]
```diff
@@ -12,6 +13,9 @@ export default class TabBar extends React.Component<TabsProps> {
       tabBarExtraContent,
       tabPosition,
       prefixCls,
+      className,
+      size,
+      type = 'line',
```
Comment (ilanus) [inline components/tabs/TabBar.tsx]: extract to `static defaultProps` ?

**Comment by zombieJ** [comment]
Updated. It's time to merge~

**Pull request merged**